### DISCUSSION TITLE
Can the .tr() function for localization detect array object?

### DISCUSSION THREAD

**Discussion by HynRidge** [nylo-core/nylo#137] [General]
Dear @agordn52 ,
Can the .tr() function for localization detect array object?
For example under `en.json` there's something like:
{
"questions": [ {"Q": "what is this?", "A": "I don't know"} ]
}
How would I reference it in the program?

**Comment by agordn52** [comment]
Hi @HynRidge,

Yes, you can 👍

Check out this section in the documentation https://nylo.dev/docs/5.20.0/localization#localizing-text

**Reply by HynRidge** [reply]
I think you misunderstand my question. instead of '{}', it's actually an array '[]'. So it's a key that holds an array of objects as it's value.

**Reply by agordn52** [reply]
Hi @HynRidge,

Sorry, it currently doesn't support arrays. 
Instead, use a `Map` for all the keys.
